sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/ProxyInvocationHandler.java

```diff
@@ -209,6 +261,69 @@ public int hashCode() {
   }
 
   /**
+   * Populate display data. See {@link HasDisplayData#populateDisplayData}.
+   */
+  private void populateDisplayData(DisplayData.Builder builder) {
+    Preconditions.checkState(canPopulateDisplayData(),
```

Comment:
Comment here why this isn't possible (so that we don't just remove the Precondition in the future).